pulp_ansible/app/tasks/collections.py

```diff
@@ -128,6 +128,7 @@ def nowhere(*args, **kwargs):
                         import_pb.increment()
 
         collection_versions = CollectionVersion.objects.filter(pk__in=collection_version_pks)
+        raise Exception('oh no!')
```

Comment:
I think this was meant to be removed

Reply:
Yes this is here for testing. I added the WIP PR because the relavence rank sorting isn't working yet. I'll try to keep the right labels on these going forward. Thank you for looking at the open PRs.